.github/workflows/build.ts.yml

```diff
@@ -14,6 +14,8 @@ jobs:
     steps:
       - name: Build and test
         uses: linz/action-typescript@v1
+        env:
+          AWS_ORG_ID: ${{secrets.AWS_ORG_ID}}
```

Comment:
how come you need this for the build and test phase? does something hit the CDK logic?

Reply:
Not sure. I was being cautious. I'll test without.